examples/colab-notebook-processing_data_using_SQl_and_local_database/colab-notebook-processing_data_using_SQl_and_local_database.ipynb

```diff
@@ -0,0 +1,387 @@
+        "\n",
+        "**Each SQL step is a separate query:**\n",
+        "\n",
+        "- The first step deletes the new table if it exists. This query only serves to make the Data Job repeatable;\n",
+        "- The second step creates the table we will be inserting data;\n",
+        "- The third step performs the described processing and inserts the new data into the customer_count_per_employee table.\n",
+        "\n",
+        "<br>\n",
+        "Run each of the following cells in order to observe the job in action.\n"
+      ],
+      "metadata": {
+        "id": "CFrCNMbL80lX"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Step 1: Drop Table"
+      ],
+      "metadata": {
+        "id": "k6DvRJegVFBB"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%vdksql\n",
+        "DROP TABLE IF EXISTS customer_count_per_employee;"
+      ],
+      "metadata": {
+        "id": "ox6Y6rYHUna2"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Step 2: Create Table"
+      ],
+      "metadata": {
+        "id": "4R4Hlkn9VWv7"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%vdksql\n",
+        "CREATE TABLE customer_count_per_employee (EmployeeId, EmployeeFirstName, EmployeeLastName, CustomerCount);"
+      ],
+      "metadata": {
+        "id": "IFGyO8VQU3Rp"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Step 3: Do the processing"
+      ],
+      "metadata": {
+        "id": "Eit_CLguVd71"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%vdksql\n",
+        "INSERT INTO customer_count_per_employee\n",
+        "SELECT SupportRepId, employees.FirstName, employees.LastName, COUNT(CustomerId)\n",
+        "FROM (customers INNER JOIN employees ON customers.SupportRepId = employees.EmployeeId)\n",
+        "GROUP BY SupportRepId;"
+      ],
+      "metadata": {
+        "id": "jWVMp4GtU7xS"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## 5. Results"
+      ],
+      "metadata": {
+        "id": "MGbX1PnhAWMb"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "After running the Data Job, we can check whether the new table was populated correctly by using the **sqlite-query** command afforded to us by the **vdk-sqlite** plugin, which we can use to execute queries against the configured SQLite database without having to set up a Data Job:"
+      ],
+      "metadata": {
+        "id": "S7RiHPv7AcvJ"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!vdk sqlite-query -q 'SELECT * FROM customer_count_per_employee'"
```

Comment:
We can use `%%vdksql` cell as well. It should provide better formatted output . We don't have to use CLI command